net/scripts/install-libssl-compatability.sh

```diff
@@ -12,4 +12,4 @@ apt-get --assume-yes install libssl-dev
 # solanalabs/rust docker image
 #
 # cc: https://github.com/solana-labs/solana/issues/1090
-apt-get --assume-yes install libssl1.1
+# apt-get --assume-yes install libssl1.1
```

Comment:
can we remove this line instead of commenting out? and are we removing since libssl 3 comes pre installed with ubuntu 22?

Reply:
sure, will do.